Automattic/wp-calypso
# Update Jetpack onboarding flow: Update instructions for logged out (new) users

#### Proposed Changes

* This PR updates the instructions on the Thank You page for logged out (new) users to make them the same as the manual installation option (for logged in users).

#### Testing Instructions



- Do any one of these
    - Open Calypso Live link below and wait for the redirect to complete
    - or boot up this PR 
        - Run `git fetch && git checkout update/jp-onboarding/instructions-for-logged-out-users`
        - Run `yarn start`
* Open a new incognito/private browser window and go to https://cloud.jetpack.com/pricing
* Purchase a product like Backup or Boost
* Complete the checkout with a new email address that does not have a WPCOM associated with it. You can add an alias to your email address with `+something` e.g. `username+jptest@something.com`
* If you want to complete the checkout for free, please check this Slack conversation: p1666675278864859-slack-C04457PNH1D. It will work for Backup and Boost
* On the Thank You page, replace `https://wordpress.com` with `http://calypso.localhost:3000` or with the Calypso Live link above
* Add `&flags=jetpack/standalone-plugin-onboarding-update-v1` query parameter to the URL to activate the feature flag.
* Confirm that you see the new instructions

#### Pre-merge Checklist






Completes 1203097885147382-as-1203159154260043/f

<img width="2270" alt="Screenshot 2022-10-26 at 4 14 26 PM" src="https://user-images.githubusercontent.com/18226415/198006945-5325820f-36ac-46f0-9f77-5810a58175f3.png">



---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - 0-as-1203159154260043
